internal/service/advancedcluster/model_advanced_cluster.go

```diff
@@ -275,6 +288,18 @@ func SchemaAdvancedConfig() *schema.Schema {
 					Type:     schema.TypeInt,
 					Optional: true,
 				},
+				"custom_openssl_cipher_config_tls12": {
+					Type:     schema.TypeSet,
+					Optional: true,
```

Comment:
Looks like we set this by default to an empty list using the `Get` in the SDK.
So we might need to support `Computed` At least in TPF where `null` != `[]`?

Reply:
For TPF we would have the following options right?
- define as computed
- add additional logic of checking if not defined in config + empty array response -> set as null.

Reply:
Correct 👍

Reply:
@EspenAlbert is this good to resolve or were there other concerns?